exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java

```diff
@@ -218,6 +230,11 @@ void getWalletUsingInvalidKey(VertxTestContext context) {
         })));
   }
 
+  private BinaryMessage getBinaryMessage() {
+    KeyPair keyPair = CRYPTO_FUNCTION.generateKeyPair();
```

Comment:
Why do we do this (generate a key pair and sign a message) if we don't check the signature anywhere?

Reply:
It's redundant could be simplified.